Source/WebInspectorUI/UserInterface/Views/CookiePopover.js

```diff
@@ -224,6 +230,14 @@ WI.CookiePopover = class CookiePopover extends WI.Popover
 
         this._secureCheckboxElement = createInputRow("secure", WI.unlocalizedString("Secure"), "checkbox", data.secure).inputElement;
 
+        this._partitionedCheckboxElement = createInputRow("partitioned", WI.unlocalizedString("Partitioned"), "checkbox", data.partitioned).inputElement;
+        let partitionKeyInputRow;
+        if (data.partitionKey) {
+            partitionKeyInputRow = createInputRow("partition-key", WI.UIString("Partition Key"), "text", data.partitionKey);
+            this._partitionKeyInputElement = partitionKeyInputRow.inputElement;
```

Comment:
we actually don't need to save this to a member variable since it's not used anywhere else, so id remove all `this._partitionKeyInputElement` and just have this be
```js
partitionKeyInputRow.inputElement.readOnly = true;
```